assignments/03-eda.md

```diff
@@ -0,0 +1,111 @@
+---
+substitutions:
+  accept_assignment: |
+    [Template repo for submission]()
+  date : 2020-09-28
+---
+
+# Assignment 3: Exploratory Data Analysis
+
+__Due:{{ date }} __
+
+{{ accept_assignment }}
+
+## Objective & Evaluation
+
+This week your goal is to do a small exploratory data analysis for two datasets of your choice.
+
+```{list-table} plot basic views of data and generate descriptive statistics and basic plots
+:header-rows: 1
+
+* - task
+    skill
+* - compute and display overall statistics
+    summarize
+* - compute and display individual statistics of a datasets
+    summarize
+* - group a data set by a variable and compute summary statics
+    summarize
+* - plot two different pairwise relationships
+    visualize
+* - interpret the statistics and plots
+  - summarize, visualize
+* - load data from at least two different file types
+  - access
+```
+
+## Choose Datasets
+
+Each Dataset must have at least three variables, but can have more. Both datasets
+must have multiple types of variables. These can be datasets you used last week,
+if they meet the criteria below.
+
+### Dataset 1 (d1)
+
+must include at least:
+- two continuous valued variables and
+- one categorical variable.
+
+
+### Dataset 2 (d2)
+
+must include at least:
+- two categorical variables and
+- one continuous valued variable
+
+
+## EDA
+
+Use a separate notebook for each dataset, name them `dataset_01.ipynb` and `dataset_02.ipynb`.
+
+For **each** dataset, in a dedicated notebook, complete the following:
+
+1. Load the data to a notebook as a `DataFrame` from url or local path, if local, include the data in your repository.
+1. Explore the dataset in a notebook enough to describe its structure use the heading `## Description`
+  - shape
+  - columns
+  - variable types
+  - overall summary statisics
+1. Write a short description of what the data contains and what it could be used for
+1. Ask and answer 4 questions using statistics, split-apply-combine, and visualizations. Make a heading for each question using a markdown cell and H2:`##`. Make sure your analyses meet the criteria in the check lists below. Interpret the answer from the analysis/plot.
+1. Describe what, if anything might need to be done to clean or prepare this data for further analysis
+
+### Dataset 1 Checklist
+
+
+
+1. Overall summary statistics grouped by a categorical variable
+1. A single statistic grouped by a categorical variable
```

Comment:
For this one do they have to use the describe()
but I have no idea what is mean by "grouped by a categorical variable" and "single statistic grouped by a categorical variable"

Reply:
We can talk later, this is public.